week-1-practise/level-1/3-syntax-error-3.js

```diff
@@ -1,6 +1,6 @@
 // There is a syntax error with this code. Fix it.
 
-function(a, b, c) {
+let addNumbers= function(a, b, c) {
```

Comment:
This absolutely works but it is not the most common way! There is another way to write this line that doesn't use an equals sign or let/var/const. Can you comment with the other solution?

Reply:
ways to write this function
1-function addNumbers(a, b, c){}
2- (a, b, c) => {}. and more concise later
However, I read here 
https://stackoverflow.com/questions/1925976/declaring-functions-in-javascript
that the first method is a good practice.